server/pages/index.js

```diff
@@ -370,11 +371,18 @@ module.exports = function() {
 			}
 		} );
 
-	app.get( '/browsehappy', function( request, response ) {
-		response.status( 200 ).render( 'browsehappy.jade', {
-			urls: generateStaticUrls( request )
+	app.get( '/browsehappy', setUpRoute, function( req, res, next ) {
+		const wpcomRe = /^https?:\/\/[A-z_-]+\.wordpress\.com$/;
+		const primaryBlogUrl = get( req, 'context.user.primary_blog_url', '' );
+		const isWpcom = wpcomRe.test( primaryBlogUrl );
+		req.context = Object.assign( {}, req.context, {
```

Comment:
Modules in `/server` tend to be more conservative on the ES6+, which is why I was playing it safe, but you know I'm all for deconstruction syntax. 😄